cumulus/parachains/common/src/xcm_config.rs

```diff
@@ -78,3 +78,59 @@ impl<Location: Get<MultiLocation>> ContainsPair<MultiAsset, MultiLocation>
 		matches!(asset.id, Concrete(ref id) if id == origin && origin == &Location::get())
 	}
 }
+
+/// Accepts an asset if it is a native asset from the system (Relay Chain or system parachain).
+pub struct ConcreteNativeAssetFromSystem;
+impl ContainsPair<MultiAsset, MultiLocation> for ConcreteNativeAssetFromSystem {
+	fn contains(asset: &MultiAsset, origin: &MultiLocation) -> bool {
+		log::trace!(target: "xcm::contains", "ConcreteNativeAssetFromSystem asset: {:?}, origin: {:?}", asset, origin);
+		let parent = MultiLocation::parent();
+		let is_system = match origin {
+			// The Relay Chain
+			MultiLocation { parents: 1, interior: Here } => true,
+			// System parachain
+			MultiLocation { parents: 1, interior: X1(Parachain(id)) } if *id < 2000 => true,
```

Comment:
I started https://github.com/paritytech/polkadot-sdk/pull/1406, so we should be able to reuse the same constant like `ParaId::from(id).is_system()` instead of `id < 2000`.

I dont want to block this PR, so if this is merged first, then I will change it in my PR and vice versa.